backend/core/src/shared/utils/get-public-email-url.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Creates a email URL object from passed url applies redirectUrl and listingId query params if they exist
+ * If they do not exist, the value will be return the email url with just the necessary token
```

Comment:
`If they do not exist, the return value will be the email url with just the necessary token`